.circleci/config.yml

```diff
@@ -88,6 +88,13 @@ commands:
       - attach_workspace:
           # Must be absolute path or relative path from working_directory
           at: ~/
+      - run:
+          name: set up node and npm install
+          command: |
+            wget https://nodejs.org/dist/v12.13.0/node-v12.13.0-linux-x64.tar.xz
+            tar -xf node-v12.13.0-linux-x64.tar.xz
+            export PATH="`pwd`/node-v12.13.0-linux-x64/bin:${PATH}"
+            npm install
```

Comment:
We already have node installed as part of `emsdk` above, right?

Hopefully we can do something like: 

 ```
source emsdk_master/emsdk_env.sh
NODE_DIR=$(dirname $EMSDK_NODE)
export PATH=$NODE_DIR:$PATH
```

Reply:
Hmm, not sure at all. Changed to that. I do see emsdk in https://github.com/emscripten-core/emscripten/blob/incoming/.circleci/config.yml#L23 , but that does not seem to help. Switched to using the above in https://github.com/emscripten-core/emscripten/pull/9989/commits/2d6fc435fa3e30d975287a1dd0598c70fcb43860 but it is not locating emsdk there.

Reply:
Sorry.. that should be "emsdk-master" with a hyphen

Reply:
Lets see if we can land this part separately.     We already run `npm install ws` in `test-upstream-sockets-chrome`.. we should probably just move that code here and remove the `ws` so it follows the package.json.

Reply:
I tried to split this off here: #10254.. hope you don't mind me being proactive here.

Reply:
Ok this part has landed now.

Reply:
Revert this?